odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,23 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
-
-Properties with a defined default value, nullable properties, and
+Non-insertable properties SHOULD be omitted from the request body.
+If they are provided, services MUST either ignore the value in the request body or fail the request.
```

Comment:
Non-breaking alternative:
```suggestion
If they are provided, services MUST ignore the value in the request body.
```

Reply:
Note: if we keep the fail part, we should qualify it with the service can only fail if the provided values do not match the service determined values.